rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -196,14 +196,15 @@ def test_parsing_forex_cache_works(
     assert inquirer._query_fiat_pair(A_EUR, A_JPY)[0] == price
 
 
+@pytest.mark.vcr()
 @pytest.mark.parametrize('use_clean_caching_directory', [True])
 @pytest.mark.parametrize('should_mock_current_price_queries', [False])
-def test_fallback_to_coingecko(inquirer):  # pylint: disable=unused-argument
+def test_fallback_to_coingecko(inquirer: Inquirer):
     """Cryptocompare does not return current prices for some assets.
     For those we are going to be using coingecko"""
-    price = inquirer.find_usd_price(EvmToken('eip155:1/erc20:0xFca59Cd816aB1eaD66534D82bc21E7515cE441CF'))  # RARI # noqa: E501
+    price = inquirer.find_usd_price(EvmToken('eip155:1/erc20:0xFca59Cd816aB1eaD66534D82bc21E7515cE441CF'), skip_onchain=True)  # RARI # noqa: E501
```

Comment:
what is this skip_onchain for?

Reply:
when coingecko rate limits the test I saw that there was an exception because the on chain oracles weren't initialized. For the inquirer fixtures we have inquirer that is the normal inquirer and inquirer_defi that initializes uniswap and uniswap v3